### ISSUE TITLE
Installation gets stuck

### ISSUE THREAD

**Issue by Gaurav11ME** [nn4pde/SPINN#1]
Hello,

I am trying to install the package. Up to pip installation everything works fine, but after that the installation remains stuck at "Installing pip dependencies". I tried manually installing mayavi and automan but the versions available are not working with python 3.8. I am using conda for installation.

**Comment by prabhuramachandran** [comment]
Yes the latest VTK breaks Mayavi, can you please use an older VTK version like so: 
`pip uninstall vtk`
`pip install vtk==9.0.1` 
and then try installing Mayavi and I believe it should work.
